universal-application-tool-0.0.1/app/views/components/FieldWithLabel.java

```diff
@@ -218,9 +230,12 @@ public ContainerTag getContainer() {
 
     ContainerTag labelTag =
         label()
-            .condAttr(!this.id.isEmpty(), Attr.FOR, this.id)
-            .withClasses(labelText.isEmpty() ? "" : BaseStyles.INPUT_LABEL)
-            .withText(labelText);
+            .attr(Attr.FOR, this.id)
+            .withCondClass(!labelText.isEmpty(), BaseStyles.INPUT_LABEL)
+            // If the text is screen-reader text, then we want the label to be screen-reader
+            // only.
+            .withCondClass(labelText.isEmpty(), Styles.SR_ONLY)
```

Comment:
(blocking) it is clearer to do `withClass(labelText.isEmpty() ? Styles.SR_ONLY : BaseStyles.INPUT_LABEL)`.

Reply:
Sure.